### PR TITLE
Feat/inline edit with error

### PR DESCRIPTION
**Pull request by scrymetto**
Now validation outside the InlineEdit is possible (+ new prop isEditing for custom behavior)

### CODE REVIEW

File: packages/inline-edit/src/interfaces.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export type InlineEditTheme = ITheme<IInlineEditTheme>;
 export interface IFieldProps<T> {
   value: T;
   onChange: (e: T) => void;
+  invalid?: boolean;
+  errorMessage?: string;
```
Comment (tundraray) [inline packages/inline-edit/src/interfaces.ts:22]: `errorMessage` can be like string or array string and I think it's better to call `error`

File: packages/inline-edit/src/InlineEdit.tsx [hunk 2]
```diff
@@ -8,25 +8,27 @@ import { getInlineEditUncontrolled } from './InlineEditUncontrolled';
 import { IInlineEditProps } from './interfaces';
 
 
-interface IInlineEditState {
-  isEditing?: boolean;
-}
-
 const getPureInlineEdit: <TFieldValue>() => FC<
```
Comment (tundraray) [inline packages/inline-edit/src/InlineEdit.tsx:11]: Can you rewrite this function as a component?

File: packages/inline-edit/src/interfaces.ts (hunk 1 above)
Comment (scrymetto) [reply packages/inline-edit/src/interfaces.ts:22]: Should I change the value of this field? I mean, 
`error?: string | string[];`

File: packages/inline-edit/src/InlineEdit.tsx (hunk 2 above)
Comment (scrymetto) [reply packages/inline-edit/src/InlineEdit.tsx:11]: yes, sure 👍

**Pull request merged**